helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/MessageProxyService.java

```diff
@@ -170,33 +158,18 @@ private void createClientProxy(Ticket ticket, HelpDeskAiSessionContext aiSession
 
     helpDeskAi.startConversation(aiSessionContext.getAiSessionKey(), aiConversation);
 
+    aiSessionContext.setIdleTimerManager(idleTimerManager);
+
     MessageProxy messageProxy = new MessageProxy();
-    messageProxy.setAgentProxyTimer(new ProxyIdleTimer(idleTicketConfig.getTimeout(),
+    messageProxy.setProxyIdleTimer(new ProxyIdleTimer(idleTicketConfig.getTimeout(),
```

Comment:
Why do you still need MessageProxy class? It seems unnecessary.

Reply:
ok